include/ola/Clock.h

```diff
@@ -222,7 +222,11 @@ class Clock {
  public:
   Clock() {}
   virtual ~Clock() {}
+  // Get the current monotonic clock if available,
+  // otherwise falls back to wall clock time.
```

Comment:
Can you define this as Doxygen, like the class comment.

Reply:
done